client/src/pages/donation/settings.js

```diff
@@ -138,38 +148,39 @@ export class DonationSettingsPage extends Component {
       <Fragment>
         <Helmet title='Manage your donation | freeCodeCamp.org' />
         <Grid>
-          <Row>
-            <Col sm={6} smOffset={3} xs={12}>
-              <Spacer size={2} />
-              <Button block={true} bsStyle='primary' href='/donate'>
+          <main>
+            <Spacer size={2} />
+
+            <FullWidthRow>
```

Comment:
```suggestion
            <FullWidthRow className='button-group'>
```

Reply:
I have gone ahead and cleaned up the layout further and added this class at more places. Thanks for pointing out.